src/clj_test_containers/core.clj

```diff
@@ -1,9 +1,14 @@
 (ns clj-test-containers.core
+  (:require [clojure.java.io :as io])
   (:import [org.testcontainers.containers
             GenericContainer]
            [org.testcontainers.utility
             MountableFile]
-           [org.testcontainers.containers BindMode]))
+           [org.testcontainers.containers BindMode]
+           [org.testcontainers.images.builder
```

Comment:
I would prefer the import statements to have one line each, instead of line breaks in them